app/js/ClosedTabRow.js

```diff
@@ -60,6 +60,7 @@ export default function ClosedTabRow(props: Props) {
           className="checkbox--td"
           onClick={_handleClickCheckbox}
           type="checkbox"
+          readOnly
```

Comment:
Can you test this out? I remember trying to address this issue a while back, but just adding `readOnly` will make the checkboxes uneditable. This needs some more involved refactoring to get rid of the React warning.

Reply:
I have no issues checking or unchecking checkboxes, just tried.